events/dvg_dua.txt

```diff
@@ -0,0 +1,3882 @@
+	on_opened_soundeffect = "event:/SFX/Events/southamerica/aristocrats"
+
+	icon = "gfx/interface/icons/event_icons/event_fire.dds"
+
+	duration = 3
+
+	immediate = {
+		if = {
+			limit = {
+				NOT = { exists = c:ENG }
+			}
+
+			create_country = {
+				tag = ENG
+				origin = this
+				state = s:STATE_MIDLANDS.region_state:DUA
+				on_created = {
+					add_modifier = {
+						name = dvg_revolutionary_militias
+						days = 720
+					}
+					#set_variable = dua_eng_secession
+					add_loyalists = {
+						value = 0.25
+						culture = cu:british
+					}
+				}
+			}
+
+			create_diplomatic_play = {
+				target_country = c:ENG
+				type = dp_rebellion
+
+				add_war_goal = {
+					holder = c:ENG
+					type = return_state
+					target_state = s:STATE_HOME_COUNTIES.region_state:DUA
+				}
+
+				add_war_goal = {
+					holder = c:ENG
+					type = return_state
+					target_state = s:STATE_WEST_COUNTRY.region_state:DUA
+				}
+			}
+		}
+
+		if = {
+			limit = {
+				exists = c:ENG
+			}
+
+			if = {
+				limit = { exists = s:STATE_LANCASHIRE.region_state:DUA }
+				s:STATE_LANCASHIRE.region_state:DUA = { 
+					set_state_owner = c:ENG
+					set_state_type = incorporated
+				}
+			}
+
+			if = {
+				limit = { exists = s:STATE_YORKSHIRE.region_state:DUA }
+				s:STATE_YORKSHIRE.region_state:DUA = { 
+					set_state_owner = c:ENG
+					set_state_type = incorporated
+				}
+			}
+
+			if = {
+				limit = { exists = s:STATE_MIDLANDS.region_state:DUA }
+				s:STATE_MIDLANDS.region_state:DUA = { 
+					set_state_owner = c:ENG
+					set_state_type = incorporated
+				}
+			}
+
+			if = {
+				limit = { exists = s:STATE_EAST_ANGLIA.region_state:DUA }
+				s:STATE_EAST_ANGLIA.region_state:DUA = { 
+					set_state_owner = c:ENG 
+					set_state_type = incorporated
+				}
+			}
+
+			s:STATE_HOME_COUNTIES = {
+				add_claim = c:ENG
+			}
+
+			s:STATE_WEST_COUNTRY = {
+				add_claim = c:ENG
+			}
+			
+		}
+
+		if = {
+			limit = {
+				has_journal_entry = je_dual_integration
+			}
+			c:ENG = {
+				set_variable = dual_integration_secession
+			}
+		}
+		else_if = {
+			limit = {
+				has_journal_entry = je_french_integration
+			}
+			c:ENG = {
+				set_variable = french_integration_secession
+			}
+		}
+	}
+	
+	option = {
+		name = dvg_dua.520.a
+		default_option = yes
+
+		c:ENG = {
+			trigger_event = { id = dvg_dua.521 days = 0 popup = yes }
+			#trigger_event = { id = dvg_dua.524 days = 0 popup = yes }
+		}
+	}
+}
+
+dvg_dua.521 = { #Glorious Revolution - English choices
+	type = country_event
+	placement = ROOT
+
+	title = dvg_dua.521.t
+	desc = dvg_dua.521.d
+	flavor = dvg_dua.521.f
+
+	event_image = {
+		video = "gfx/event_pictures/southamerica_aristocrats.bk2"
+	}
+
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	on_opened_soundeffect = "event:/SFX/Events/southamerica/aristocrats"
+
+	icon = "gfx/interface/icons/event_icons/event_fire.dds"
+
+	duration = 3
+
+	option = {
```

Comment:
Adding laws has a tendency to cause revolts and civil wars due to dissatisfied IGs(for AI), it might be a good idea to add the dvg_approval_buff modifier to England for 10 years or so. Especially on the Republican side, given it skips straight to universal suffrage.